src/test/msgr/perf_msgr_server.cc

```diff
@@ -73,6 +73,7 @@ class ServerDispatcher : public Dispatcher {
   ServerDispatcher(int threads, uint64_t delay): Dispatcher(g_ceph_context), think_time(delay),
     op_tp(g_ceph_context, "ServerDispatcher::op_tp", "tp_serv_disp", threads, "serverdispatcher_op_threads"),
     op_wq(30, 30, &op_tp) {
+    require_authorizer = false;
```

Comment:
why?

Reply:
@tchaikov .  They only for performance tests. Like https://github.com/ceph/ceph/pull/24678 to make it simple to run.